docs/source/Installation/Solcore_on_Windows.md

```diff
@@ -1,83 +1,93 @@
 Solcore on Windows 10
 =====================
-The Python part of Solcore, which is the majority, should work under Windows with no problems. However, we have found a lot trouble trying to make the parts and complements that require compilation to work properly (the PDD solver, S4, SMARTS and SPICE). For those who need to use those tools, please follow the instructions below. 
+The Python part of Solcore, which is the majority, should work under Windows with no problems. However, we have had some
+trouble trying to make the parts and complements that require compilation to work properly (the PDD solver, S4, SMARTS 
+and SPICE). For those who need to use those tools and encounter issues on Windows, please follow the instructions below. 
+
+Note that in principle, out of the packages listed above (the PDD solver, S4, SMARTS and SPICE), all can be
+installed on Windows **except S4**, which to date we have not been able to produce Windows installation instructions for. 
+To compile the PDD, you need to do [a few extra steps](compilation.md) to get a suitable compiler working.
+S4 is only required to use the rigorous coupled-wave analysis (RCWA) functionality in Solcore, used to calculate diffraction
+from periodic structures (e.g. diffraction gratings, photonic crystals). Thus, if you do not need this functionality, you 
+can stick with Windows.
 
 Installing Solcore
 ------------------
-(thanks Phoebe!)
 
-After a lot of effort, we have Solcore fully up and running in Windows 10... more or less, using the Ubuntu shell that comes with Windows 10. To install Solcore there, follow this steps:
+It is possible to run all the parts of Solcore from a Windows 10 environment without using a virtual machine or dual-booting a UNIX operating
+system (e.g. Ubuntu). This can be done by using the Ubuntu shell that comes with Windows 10. **It should be noted that this Windows
+Subsystem on Linux (WSL) does not support graphical applications, so it is purely a command line environment.** This can be 
```

Comment:
I'm not sure I understand this. Cannot you set, for example VS Code, and use it normally (specific instructions [here](https://code.visualstudio.com/docs/remote/wsl-tutorial))? Granted it is more convoluted but still.

Reply:
I did not know about this, I don't think this was an option when I was using the WSL. I will test out if this works and depending on what I find I will edit this part and link to this tutorial.

Reply:
Wow, this works! Just successfully ran examples using S4 from a Windows environment by connecting to WSL Ubuntu through VSCode (including plots). All the tests pass too running from the VSCode terminal connected to the WSL. Actually very impressed with this, it's very well-integrated, they've really improved this functionality a lot since I used it a few years ago. I will update this section of the instructions to link to the tutorial on the VisualStudio website and some other details.

Reply:
And it is also possible to do this [in PyCharm](https://www.jetbrains.com/help/pycharm/using-wsl-as-a-remote-interpreter.html)! Glad they sorted this out just at the same time I stopped using a Windows computer 😂